app/jobs/process_funding_csv_job.rb

```diff
@@ -0,0 +1,8 @@
+class ProcessFundingCsvJob < ApplicationJob
```

Comment:
I think this job will also need to be added somewhere here https://github.com/DFE-Digital/register-trainee-teachers/pull/3930/files#diff-b03c967c04be863ba7c78e181a3bf217f068b9bac69ccef018a9ebbee5bf3c80R19 right after the form is submitted and the `FundingUpload` record is created.